misk-core/src/main/kotlin/misk/logging/Logging.kt

```diff
@@ -2,14 +2,27 @@ package misk.logging
 
 import misk.sampling.Sampler
 import mu.KLogger
-import mu.KotlinLogging
 import org.slf4j.MDC
 import org.slf4j.event.Level
 
+@Deprecated(
+  "Use wisp-logging",
+  ReplaceWith(
+    "Tag",
+    "wisp.logging.Tag",
+  )
+)
 typealias Tag = Pair<String, Any?>
 
+@Deprecated(
+  "Use wisp-logging",
+  ReplaceWith(
+    "getLogger<T>()",
+    "wisp.logging.getLogger",
+  )
+)
 inline fun <reified T> getLogger(): KLogger {
-  return KotlinLogging.logger(T::class.qualifiedName!!)
+  return wisp.logging.getLogger<T>()
```

Comment:
I realise I've taken a different approach locally - instead of deprecating and calling them `wisp.logging.*` I've kept the same package name and moved them to a different module. I think this will reduce disruption. Thoughts on changing package vs keeping it the same?

Reply:
Yes.  I want to keep the wisp structure clean from any misk modules.  So for most things I've been wrapping the wisp mosules in the misk packages.  The deprecations are because I could not do it here.